convert2rhel/toolopts.py

```diff
@@ -191,7 +192,15 @@ def _register_options(self):
         self._shared_options_parser.add_argument(
             "--eus",
             action="store_true",
-            help="Automatically recognize the system as eus, utilizing eus repos.",
+            help="Automatically recognize the system as eus, utilizing eus repos."
+            " 8.6 systems do not require this option as they are recognized as eus automatically."
```

Comment:
Since we will drop 8.6 support in the next release, does this text is still accurate? I don't think so, right?

Reply:
Ah good point - I can remove it